Makefile

```diff
@@ -1,10 +1,9 @@
 # Go parameters
-GOCMD=go
-GOBUILD=$(GOCMD) build
-GOCLEAN=$(GOCMD) clean
-GOTEST=$(GOCMD) test
-GOPATH=$(shell cd ../../../..; pwd)
-GOPACKAGES=$(shell GOPATH=`cd ../../../..; pwd` go list ./... | grep -v /vendor/ | sed 's/^_//')
```

Comment:
I agree with all your change above except this one `| sed 's/^_//'`.
Could you give me a reason that why you eliminate this part?
Thanks a lot. @qingyunha

Reply:
Maybe I am wrong. I don't see any package name starts with '_'.

Reply:
So could you keep this part as it was.

Reply:
Ok. updated